portfolio/src/main/webapp/slide.css

```diff
@@ -1,4 +1,8 @@
 /* styles for the slideshow */
+img {
+  display: none;
+}
+
```

Comment:
I wouldn't recommend adding a blanket style like this to all img tags because you don't want your styles to be overly broad; every single img you add will have this style now. Try making the qualifier more specific.

```
.slides img.pic { ... } 
```